### PR TITLE
[malwarebytes-254] Updated Domain to threatdown.com in Usage Analytics

### PR DESCRIPTION
**Pull request by rskumar-mwb**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py [hunk 1]
```diff
@@ -1414,7 +1414,7 @@ def send_usage_data(auth_token, account_id, client_id, msg_type, USE_SSL):
         "integration_code": "TA-CX",
         "integration_name": "Cortex XSOAR",
         "integration_app": "Malwarebytes",
```
Comment (mmhw) [inline Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py:1416]: `integration_app` should still remain `Malwarebytes`?

**Review by mmhw — COMMENTED** [review]
Hi @rskumar-mwb,
Thank you for your contribution!
Good work :)

In line 1456 it seems necessary to change from `Malwarebytes` to `ThreatDown`.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

File: Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py (hunk 1 above)
Comment (rskumar-mwb) [reply Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py]: yeah, that should be threatdown. I have made the changes, thanks for pointing out.

**Comment by mmhw** [comment]
Hi @rskumar-mwb,
Did you test this version on an XSOAR machine and did everything work as expected?

Thanks again!

**Comment by rskumar-mwb** [comment]
@mmhw Yes, its tested, and everything works fine.

**Pull request merged**